man/man8/zpoolconcepts.8

```diff
@@ -93,6 +93,59 @@ compromised.
 The minimum number of devices in a raidz group is one more than the number of
 parity disks.
 The recommended number is between 3 and 9 to help increase performance.
+.It Sy draid , draid1 , draid2 , draid3
+A distributed version of raidz with integrated hot spares that allows for
+faster resilvering.
+A draid vdev is constructed from multiple internal raidz groups, each with D
+data devices and P parity devices, which are distributed over the children.
+This allows all of the children to participate when resilvering.
+As a general rule the more devices in a draid the less time required to
+resilver to a distributed spare.
+Furthermore, a narrower stripe width (D+P) will improve resilver performance
+at the expense of total usable capacity.
+.Pp
+Like raidz a draid can have single-, double-, or triple-parity.  The
+.Sy draid1 ,
+.Sy draid2 ,
+and
+.Sy draid3
+types can be used to specify the parity level.
+The
+.Sy draid
+vdev type is an alias for
+.Sy draid1 .
+.Pp
+A draid with N disks of size X, D data disks, P parity level, and H distributed
+hot spares can hold approximately (N-H)*(D/(D+P))*X bytes and can withstand P
+device(s) failing before data integrity is compromised.
+The minimum number of devices in a draid is P+2.
+.It Sy draid[<parity>][:<data>d][:<children>c][:<spares>s][:<iterations>i]
+When a non-default draid configuration is required it can be specified by
+appending one or more of the following optional arguments to the
+.Sy draid
+keyword.
+.Pp
+.Em children
+- The expected number of children.
+Useful as a cross-check when listing a large number of devices, an error is
+returned when the provided number of children differs.
+.Pp
+.Em data
+- The number of data devices per redundancy group.  Defaults to a maximum
+value of 8.
```

Comment:
For the `data` section, I would add your language from the PR description, plus an table to quantify how the data value can affect rebuild speeds and capacity.  I put together a table using your rebuild speed data and added my own data from testing:
```suggestion
value of 8. Wider stripes require reading more disks in order to reconstruct
the missing data but yield a larger usable capacity. Conversely, narrower
stripes require reading fewer disks and rebuild more quickly at the expense
of capacity.  Below are some measurements found in one setup (your mileage
will vary):
.Pp
.TS
tab(@);
c s s c c n .
data@Rebuild slowdown compared to data=1@Capacity increase compared to data=1
1@0%@0%
2@44%@33%
3@62%@49%
4@74%@60%
5@81%@64%
6@85%@70%
7@88%@73%
8@90%@78%
.TE
.Pp
```

Reply:
I've updated `zpoolconcepts.8` with a better explanation of dRAID and the options.  I opted to leave out the table since it felt a bit out of place here, perhaps a better place for this would be the OpenZFS dRAID page which I intend to update.